test/controllers/FactCheckConfirmControllerSpec.scala

```diff
@@ -42,7 +46,10 @@ class FactCheckConfirmControllerSpec extends ControllerBaseSpec with GuiceOneApp
     val errorHandler: ErrorHandler = injector.instanceOf[ErrorHandler]
 
     val view: fact_check_complete = injector.instanceOf[fact_check_complete]
+    val approvalProcessSummary: ApprovalProcessSummary = ApprovalProcessSummary(id, "title", LocalDate.now, ApprovalStatus.Published)
```

Comment:
Is approvalProcessSummary being used?

Reply:
probably not now - I changed what was coming back from external-guidance, but missed removing this